### PR TITLE
Add Voucher thank you page

### PR DESCRIPTION
**Pull request by nlindblad**
**Voucher Sixday+**:

screen shot 2016-08-19 at 11 36 43

**Voucher Sixday**:

screen shot 2016-08-19 at 11 39 14

### CODE REVIEW

File: app/views/checkout/thankyou.scala.html [hunk 1]
```diff
@@ -63,6 +63,10 @@ <h4 class="steps__title">We’ll send you an email confirming your subscription
                 @if(plan.products.seq.contains(Delivery)) {
                     <h4 class="steps__title">Your first paper will be delivered on @startDate. </h4>
                 }
+                @if(!plan.products.seq.contains(Digipack) && !plan.products.seq.contains(Delivery)) {
```
Comment (nlindblad) [inline app/views/checkout/thankyou.scala.html]: This will hopefully be simpler to determine in the future.

Comment (tomverran) [inline app/views/checkout/thankyou.scala.html]: yes!

<sub><sub>hopefully</sub></sub>

File: app/views/checkout/thankyou.scala.html [hunk 2]
```diff
@@ -63,6 +63,10 @@ <h4 class="steps__title">We’ll send you an email confirming your subscription
...
+                    <h4 class="steps__title">You will then be receive your personalised book of voucher.</h4>
```
Comment (tomverran) [inline app/views/checkout/thankyou.scala.html]: You will then be receive?

**Comment by tomverran** [comment]
👍

**Pull request merged**